src/App/App.tsx

```diff
@@ -1,9 +1,40 @@
 import * as React from 'react';
-import {MainRoutes} from './MainRoutes'
+import { Navigation } from '../shell/navigation/index';
+import { IRoute } from '../shell/navigation/navigationItem/NavigationItem';
+import { Finance } from '../Modules/Finance';
```

Comment:
We should use ../shell/navigation/ instead of ../shell/navigation/index
Also maybe the better idea to import IRoute in ../shell/navigation/index, it allows us to use only one import, like: 
import { Navigation, IRoute } from '../shell/navigation'

Reply:
Done